webapi/webapi-webgl-khronos-tests/webgl/khronos/resources/js-test-pre.js

```diff
@@ -20,6 +20,25 @@
 ** TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 ** MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
 */
+var caseName = document.title;
```

Comment:
It's only for WTS. To save subcases' results into result json file, WTS uses document titile suffixing with '/' and index as the subcase's name.